examples/04-Exporting-ranking-models.ipynb

```diff
@@ -70,9 +70,9 @@
      "name": "stderr",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    train_path = os.path.join(DATA_FOLDER, "train", "*.parquet")
</pre><p>Did you test that? <code>*.parquet</code> was not working for me what's why I had to do <code>"part.0.parquet"</code> instead. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/414/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Yes the unittest passed and it worked for me